File: HelloWorld/results/HelloWorld.java
import java.util.HashMap;

class TestClass {
    public void testMethod() throws Exception
    {
        Integer value = 1 + 2 * 3 - 4;
        HashMap<String, Integer> map_ = new HashMap<String, Integer>();
        map_.put("a", 5);
        map_.put("b", 6);
        String text = "Hello world! value = " + value + ", map[a] = " + map_.get("a");
        System.out.println(text);
    }
}

class Program {
    public static void main(String[] args) throws Exception {
        try {
            new TestClass().testMethod();
        } catch (Exception err) {
            System.out.println("Exception: " + err.getMessage());
        }
    }
}

File: HelloWorld/results/HelloWorld.php
<?php

class TestClass {
    function testMethod() {
        $value = 1 + 2 * 3 - 4;
        $map_ = array(
          "a" => 5,
          "b" => 6,
        );
        $text = "Hello world! value = " . $value . ", map[a] = " . $map_["a"];
        print($text . "\n");
    }
}

try {
    $c = new TestClass();
    $c->testMethod();
} catch (Exception $err) {
    echo 'Exception: ' . $err->getMessage() . "\n";
}